NumLib/Fem/ShapeMatrixPolicy.h

```diff
@@ -58,19 +58,23 @@ struct EigenFixedShapeMatrixPolicy
     template <int N>
     using VectorType = typename ::detail::EigenMatrixType<N, 1>::type;
 
+    template <int N>
+    using RowVectorType = typename ::detail::EigenMatrixType<1, N>::type;
+
     template <int N, int M>
     using MatrixType = typename ::detail::EigenMatrixType<N, M>::type;
 
     using NodalMatrixType = MatrixType<ShapeFunction::NPOINTS, ShapeFunction::NPOINTS>;
     using NodalVectorType = VectorType<ShapeFunction::NPOINTS>;
+    using NodalRowVectorType = RowVectorType<ShapeFunction::NPOINTS>;
     using DimNodalMatrixType = MatrixType<ShapeFunction::DIM, ShapeFunction::NPOINTS>;
     using DimMatrixType = MatrixType<ShapeFunction::DIM, ShapeFunction::DIM>;
     using GlobalDimNodalMatrixType = MatrixType<GlobalDim, ShapeFunction::NPOINTS>;
     using GlobalDimMatrixType = MatrixType<GlobalDim, GlobalDim>;
 
     using ShapeMatrices =
         NumLib::ShapeMatrices<
-            NodalVectorType,
+            NodalRowVectorType,
```

Comment:
It's surprising to me that this line is apparently all that had to be changed in order to transpose the shape matrices. Nice.